providers/tokenbucket/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/grpc-ecosystem/go-grpc-middleware/providers/tockenbucket/v2
+
+go 1.15
+
+require (
+	github.com/grpc-ecosystem/go-grpc-middleware/v2 v2.0.0-rc.2
```

Comment:
You need to manually replace `v2.0.0-rc.2` with latest commit with your interface change and run `go mod tidy` to make tests and this module working 🤗

Reply:
This won't work if the latest commit is on my local computer? :thinking:

Reply:
No. It's because Go Modules is...... special (: It kind of makes sense - they try to give you latest "stable" version. If people do sem ver properly this would be latest minor + patch version of the major version you use.

Definitely not latest commit - so you need to pin to latest commit manually

Reply:
what is the right semantic here guys?
```
 go mod tidy
go: github.com/grpc-ecosystem/go-grpc-middleware/v2@v2.0.0-rc.2-e0ea420: reading github.com/grpc-ecosystem/go-grpc-middleware/go.mod at revision v2.0.0-rc.2-e0ea420: unknown revision v2.0.0-rc.2-e0ea420
```
I have tried  also removing the version before. :grin:

Reply:
should be done now